lib/honeybadger/plugins/rails.rb

```diff
@@ -68,6 +69,19 @@ def self.source_ignored?(source)
           end
         end
       end
+
+      Plugin.register :rails do
+        requirement { config.load_plugin_insights?(:rails_metrics) && defined?(::Rails.application) && ::Rails.application }
+
+        execution do
+          ::ActiveSupport::Notifications.subscribe(/(process_action|send_file|redirect_to|halted_callback)\.action_controller/, Honeybadger::ActionControllerSubscriber.new)
+          ::ActiveSupport::Notifications.subscribe(/(write_fragment|read_fragment|expire_fragment)\.action_controller/, Honeybadger::ActionControllerCacheSubscriber.new)
+          ::ActiveSupport::Notifications.subscribe(/^render_(template|partial|collection)\.action_view/, Honeybadger::ActionViewSubscriber.new)
+          ::ActiveSupport::Notifications.subscribe("sql.active_record", Honeybadger::ActiveRecordSubscriber.new)
+          ::ActiveSupport::Notifications.subscribe("process.action_mailer", Honeybadger::ActionMailerSubscriber.new)
+          ::ActiveSupport::Notifications.subscribe(/(service_upload|service_download)\.active_storage/, Honeybadger::ActiveStorageSubscriber.new)
+        end
+      end
```

Comment:
Would it be worth adding the [unpermitted_parameters.action_controller](https://guides.rubyonrails.org/active_support_instrumentation.html#unpermitted-parameters-action-controller) here as well?

I was thinking of adding a cache hit/miss dashboard widget. There are the view fragment caching events that we subscribe to, but I wonder if we should also include the active_support cache events as well.